pkg/kubelet/remote/remote_runtime_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remote
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	internalapi "k8s.io/kubernetes/pkg/kubelet/apis/cri"
+	apitest "k8s.io/kubernetes/pkg/kubelet/apis/cri/testing"
+	fakeremote "k8s.io/kubernetes/pkg/kubelet/remote/fake"
+)
+
+const (
+	defaultConnectionTimeout = 15 * time.Second
+)
+
```

Comment:
This file seems to test the fake itself. Move into `remote/fake`?

Reply:
The file is aiming to test remote package, but here are only a basic start. will add testings in following PRs.